pyproject.toml

```diff
@@ -35,7 +34,7 @@ dependencies = [
     "sentence-stream==1.1.0",
 ]
 description = "Home Assistant cloud integration by Nabu Casa, Inc."
-license = {text = "GPL v3"}
+license = "GPL-3.0-only"
```

Comment:
Alternatively this could be `GPL-3.0-or-later` though the classifier would suggest `3.0-only`.